lib/puppet/functions/simplib/deprecation.rb

```diff
@@ -32,8 +29,6 @@ def deprecation(key, message)
       message = "#{message} at #{file}:#{line}"
     end
 
-    if ENV['SIMPLIB_LOG_DEPRECATIONS'] == 'true'
-      Puppet.deprecation_warning(message, key)
-    end
+    Puppet.deprecation_warning(message, key)
```

Comment:
What do they do to turn off the deprecation warnings?

Reply:
they don't, they change their code to call the correct one, since this is the last release having these in here I felt being more heavy handed with it was justified and reasonable.